### PR TITLE
26 reset css 2

### LINKED ISSUE #26: reset css 2
We need to implements a reset css new

### PR DESCRIPTION
**Pull request by Tim007007**

### CODE REVIEW

File: Css/reset.css [hunk 1]
```diff
@@ -1,6 +1,89 @@
...
+}
+
+a  {
+    color: inherit;
+    text-decoration: none;
+}
+
+button {
+    color: inherit;
+    font-size: 10px;
```
Comment (Maiia2023) [inline Css/reset.css]: font-size: inherit;

File: Css/reset.css [hunk 2]
```diff
@@ -1,6 +1,89 @@
...
+}
+
+ul,
+ol {
     list-style: none;
-}
\ No newline at end of file
+}
+
+a  {
```
Comment (Maiia2023) [inline Css/reset.css:25]: remove 1 space after a

File: Css/reset.css [hunk 3]
```diff
@@ -1,6 +1,89 @@
...
+}
+
+/*
+button:
+color: зависимый
+шрифт: зависимыый
+бэкграунд: отсутствует
+отступы внутренние: отсутствует
+*/
+
```
Comment (Maiia2023) [inline Css/reset.css]: better to remove it from this file. If you want to keep it for yourself make a copy in _temp folder.

**Review by Maiia2023 — APPROVED** [review]

**Comment by Tim007007** [comment]
i'll do it

**Pull request closed without merging**